README.md

```
@@ -1,19 +1,106 @@
+  <tr>
+    <th>Framework / Library</th>
+    <th>Styling</th>
+    <th>State & Data</th>
+    <th>Routing</th>
+    <th>UI / Interaction</th>
+    <th>3D / Geo / Calendar</th>
+    <th>Dev Tools</th>
+  </tr>
+  <tr>
+    <td>
+      <img src="https://img.shields.io/badge/React-61DAFB?logo=react&logoColor=white" /><br />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)